### ISSUE TITLE
running api tests that require an email server does not work with docker

### ISSUE THREAD

**Issue by individual-it** [owncloud/ocis#6314]
## Describe the bug

When running tests using docker as described here: https://owncloud.dev/ocis/development/testing/#testing-with-test-suite-in-docker one cannot run tests that require a email server.

Running the email server (inbucket) should be default in the docker environment

**Comment by amrita-shrestha** [comment]
I think inbucket has not been setup in docker yet

**Comment by individual-it** [comment]
@amrita-shrestha yes that is the main issue, it should be setup and enabled by default
